src/main/java/yasp/Main.java

```diff
@@ -2,35 +2,61 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.FileInputStream;
 import java.io.OutputStream;
 import java.net.InetSocketAddress;
 import com.sun.net.httpserver.HttpExchange;
 import com.sun.net.httpserver.HttpHandler;
 import com.sun.net.httpserver.HttpServer;
     
 public class Main {
+    
+    public static void main(String[] args) {
+	try { 
+                       
+	    if (args.length > 0 && args[0].equals("--file")) { System.exit(parse(args[1])); }
```

Comment:
The current code only has the server case.  i assume you added these additional startup methods to make testing easier.  Previously, I just started the server and then used curl to POST a replay to it.  Up to you if you want to keep them around.